spec/core/Main.savi

```diff
@@ -27,4 +27,5 @@
       Spec.Run(Savi.Numeric.Spec).new(env)
       Spec.Run(Savi.Platform.Spec).new(env)
       Spec.Run(Savi.String.Spec).new(env)
+      Spec.Run(Savi.BCD.U64.Spec).new(env)
```

Comment:
There are so many specs here that I have given up on giving them a meaningful order, so they are ordered alphabetically - can you move this line to the appropriate place in the sorted list?

After taking into account the suggested name change I mentioned in my other comment, of course.